docs/wiki/UtilitiesTestModuleLocally.md

```diff
@@ -20,15 +20,14 @@ You can find the script under `/utilities/tools/Test-ModuleLocally.ps1`
 If the switch for pester tests (`-PesterTest`) was provided the script will
 1. Invoke the global module test for the provided template file path and run all tests for it
 
-If the switch for either the validation test (`-ValidationTest`) or deployment test (`-DeploymentTest`) was provided alongside a hashtable for the token replacement (`-ValidateOrDeployParameters`), the script will
-1. Search all parameter files for the given module template
-1. Craft a dictionary to replace all tokens in these parameter files with actual values. This dictionary will consist
+If the switch for either the validation test (`-ValidationTest`) or deployment test (`-DeploymentTest`) was provided alongside a HashTable for the token replacement (`-ValidateOrDeployParameters`), the script will
+1. either fetch all parameter files of the module's parameter folder (default) or use a specify parameter file you can provide via the `parameterFilePath` parameter instead.
+1. Create a dictionary to replace all tokens in these parameter files with actual values. This dictionary will consist
    - of the subscriptionID & managementGroupID of the provided `ValidateOrDeployParameters` object,
    - add all key-value pairs of the `-AdditionalTokens` object to it,
-   - and optionally also add all key-value pairs specified in the `settings.json`'s `parameterFileTokens` object if the `-SkipParameterFileTokens` parameter was not set
-1. It replaces all tokens in the parameter files as per the object created in the previous step
+   - and optionally also add all key-value pairs optionally specified in the `settings.json`'s `parameterFileTokens` object
```

Comment:
and optionally also add all key-value pairs specified in the `settings.json`'s `parameterFileTokens` object

Reply:
updated